### ISSUE TITLE
Add charts to custom reports

### ISSUE THREAD

**Issue by fvanderhaegen** [trimble-oss/dba-dash#909]
Hi,

I don't know if this is possible but a great feature would be the possibility to add charts to custom reports.
Now we see a list of data in custom reports but sometimes a chart would a better (more usefull) way to display the info.

**Comment by DavidWiseman** [comment]
Yes - I agree.  I would like to add the option for charts and it's something I've thought about but left it out of scope for the initial implementation.  I'm not sure when, but it's likely to get added at some point.
